### ISSUE TITLE
Problem with connection when using proxy and VPN

### ISSUE THREAD

**Issue by syleniaa** [fmd-project-team/FMD#1046]
Problem with connection when using proxy and VPN. First i put my proxy and it was working OK, but i needed a VPN because my proxy blocked me some sites , at first everything worked OK, but after a time the application doesn't connect anymore. I tried turning off the vpn and just configuring the proxy and nothing.

**Comment by Tmp341** [comment]
You removed proxy settings?

**Comment by syleniaa** [comment]
no, i dont remove them. i use psiphon vpn, and what i do it's use as proxy in fmd is 127.0.0.1 and the port that psiphon tells me to use. at first it connected ok like that but after a time doesnt connect anymore

**Comment by Tmp341** [comment]
Try to remove proxy settings from FMD.

**Comment by syleniaa** [comment]
i just tried and it keeps without connection. i dont't know what happened because a few days ago the application was working fine with the proxy and the vpn, and before that i have only used it with proxy but not with the vpn. is there a way that i can see what is the response of the requests made by the app?? i would like to see wha error with the connectio i s giving me

**Comment by syleniaa** [comment]
![Image e](https://user-images.githubusercontent.com/16405194/85938403-25d0a200-b8db-11ea-95ea-ffaf11c3e2d4.png)
this image is what is happening now , and when i try to get any manga info it gave me an empty response but it's not giving me any error

![Image 2w](https://user-images.githubusercontent.com/16405194/85938429-5e707b80-b8db-11ea-91f8-d0c21eb8ab33.png)

i though that maybe the site was blocked but i can access in the browser to all manga websites that i use

**Comment by syleniaa** [comment]
i use the vpn because the images from ninemanga doesnt load in th browser and with the vpn it does. but even without the vpn the application used to show me all manga info and the chapters to download, but now it just doesn't connect to internet with and without proxy and vpn.

**Comment by syleniaa** [comment]
this is one of the websites that i use, it's the one that i use the most and for the one that i have to use the vpn for the images to load. i am using the proxy and the vpn and the website is loading fine ![Image 35](https://user-images.githubusercontent.com/16405194/85938689-73e6a500-b8dd-11ea-98ca-39d8c42377b2.png)

**Comment by syleniaa** [comment]
now i am testing with other websites i the app and i think is a problem with http://es.ninemanga.com/, i tried getting the info of a random manga from another website and is working fine, why would happen this if http://es.ninemanga.com/ is loading well in the browser??
![Image d6](https://user-images.githubusercontent.com/16405194/85938756-f8d1be80-b8dd-11ea-9e09-bd6ef251a049.png)

**Comment by syleniaa** [comment]
look what is doing now, it giving me N/A to any manga from http://es.ninemanga.com/ and a few moments ago at least it was giving me the manga name and before that, a few days ago it was working fine
![Image g7](https://user-images.githubusercontent.com/16405194/85938824-975e1f80-b8de-11ea-9dab-eb658e22a8e0.png)

**Comment by syleniaa** [comment]
now that i now the problem is with http://es.ninemanga.com/ do i create a new issue or leave it here???

**Comment by syleniaa** [comment]
i am trying and the app only conects when the vpn is on, but to another websites , not http://es.ninemanga.com/

**Comment by Tmp341** [comment]
VPN might be blocked by es.ninemanga part (IP adress most likely).

It is not FMD's problem, it is with your vpn and ninemanga, so i'm closing this.

**Comment by syleniaa** [comment]
a question , how can i still access es.ninemanga in the browser with the vpn??

**Comment by Tmp341** [comment]
VPN blocks whole traffic from your pc. You need to access with another device, or disable it for that site.
